juniper_official/chassis/check-fan-state-rpm.rule

```diff
@@ -24,8 +24,8 @@ healthbot {
              */			
             sensor fan-netconf {
                 iAgent {
-                    file fanEVOTable.yml;
-                    table fanEVOTable;
+                    file fanTable.yml;
```

Comment:
Why new file?

Reply:
Updated file name to same as old name (fanEVOTable.yml).